### PR TITLE
Enable Wifi on ePC2

### PR DESCRIPTION
**Pull request by AntonSchmied**
This deploys a config file for the Networkmanager to read from and enables the latter. For now the SSID is fixed and cannot be changed. It is enough to start testing the functionality though.

### CODE REVIEW

File: build-tools/epc2/create-update/createEPC2UpdateDockerCommands.sh [hunk 1]
```diff
@@ -21,7 +21,47 @@ install_packages() {
     
     echo "en_US.UTF-8 UTF-8" > /overlay-fs/etc/locale.gen
     /bin/arch-chroot /overlay-fs locale-gen
+}
+
+setup_wifi() {
+    (
```
Comment (hhoegelo) [inline build-tools/epc2/create-update/createEPC2UpdateDockerCommands.sh:27]: [did you find | can you explain] the reason for the braces?

Comment (AntonSchmied) [reply build-tools/epc2/create-update/createEPC2UpdateDockerCommands.sh]: no ... so i kicked them out and got rid of the quotes, which were also unnecessary in out case.
tested, works fine.

File: build-tools/epc2/create-update/createEPC2UpdateDockerCommands.sh [hunk 2]
```diff
@@ -21,7 +21,46 @@ install_packages() {
     
     echo "en_US.UTF-8 UTF-8" > /overlay-fs/etc/locale.gen
     /bin/arch-chroot /overlay-fs locale-gen
+}
+
+setup_wifi() {
+
+    cat <<- ENDOFHERE > /overlay-fs/etc/NetworkManager/system-connections/C15.nmconnection
```
Comment (hhoegelo) [inline build-tools/epc2/create-update/createEPC2UpdateDockerCommands.sh:28]: Looks good - but I have still problems understanding the bash heredoc syntax: piping into the file is done between declaring the delimiter and the actual here-doc? WTF? How does the interpreter decide where the doc starts? After a new line? Suspicious, in my opinion.
Anyway, glad you found a way of having readable AND working code.

**Review by hhoegelo — APPROVED** [review]

File: build-tools/epc2/create-update/createEPC2UpdateDockerCommands.sh (hunk 2 above)
Comment (AntonSchmied) [reply build-tools/epc2/create-update/createEPC2UpdateDockerCommands.sh:28]: Best description I found was here https://linuxize.com/post/bash-heredoc/

**Pull request merged**